src/components/PostList/PostList.tsx

```diff
@@ -1,5 +1,13 @@
 import React from 'react';
+import { Post } from '../../types/Post';
+import { PostInfo } from '../PostInfo';
 
-export const PostList: React.FC = () => (
-  <>Put the list here</>
+type Prop = {
```

Comment:
Up to you, but you can use the interface syntax instead of 'type' in components, not only in .ts files